config/files/server/usr/etc/ssh/sshd_config.d/99-hardening.conf

```diff
@@ -1,3 +1,9 @@
+# Secure sshd Template
+
+# If you want to add configurations, instead of editing this file,
+# create another one in the same directory.
```

Comment:
This is almost good, but it should also say that the new file needs to be alphabetically later.

Also, can you put this in the FAQ instead?

Reply:
yes, I suppose having a central location is good. I came to the same idea but will not add this there, just add this warning (And I think we should add it on every such file?)

then add a FAQ entry on how to add supplementary and overwrite configs.

Reply:
Just the FAQ please to avoid redundancy